doc/client_lua_api.md

```diff
@@ -782,6 +782,41 @@ An interface to manipulate minimap on client UI
 * `set_shape(shape)`: Sets the minimap shape. (0 = square, 1 = round)
 * `get_shape()`: Gets the minimap shape. (0 = square, 1 = round)
 
+### Camera
+An interface to get or set information about the camera and cameranode.
+Please do not try to access the reference until a second or so after the client
```

Comment:
`or so`. Accuracy could be better here.
Just add a notice that the reference will be nil until the camera is initialized.